src/PDO/Peachpie.Library.PDO/PDOMySQLWrapper.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace Peachpie.Library.PDO
+{
+    //TODO: Probably a similar class is needed for the other DBMS
+    class PDOMySQLWrapper
+    {
+        public static string convertDSN(string dsn)
+        {
+            if (dsn == null || dsn.Length == 0)
+            {
+                return dsn;
+            }
+
+            dsn = dsn.Replace("host", "Host");
+            dsn = dsn.Replace("port", "Port");
+            dsn = dsn.Replace("dbname", "Database");

```

Comment:
we have the idea, of course we cannot change "dbname" to "Database" in connection string, that would be an unpleasant sideeffect.